src/responseProducer/parser/file.parser.middleware.ts

```diff
@@ -0,0 +1,26 @@
+import { DataObjectType } from '../../mongo-rabbit/src/paramTypes';
```

Comment:
Interesting structure of files. We'll talk about it before approval of the changes